system/ee/language/english/fieldtypes_lang.php

```diff
@@ -49,7 +49,7 @@
 
     'rel_ft_limit' => 'Maximum entries',
 
-    'rel_ft_limit_desc' => 'Maximum number of entries to show in relationship field.<br><i>Leave blank to allow all entries.</i>',
+    'rel_ft_limit_desc' => 'Maximum entries displayed in dropdown. <br><i>Leave blank to allow all entries.</i>',
```

Comment:
Maybe revamp this to something like:  'Sets the number of entries initially displayed in the field's dropdown. A high number of entries can increase the load time of the field.'

Reply:
I'd not include 'initial' because AFAIK this also applies to AJAX search results

Reply:
I wish you could actually scroll and have more show up, so 'initially' would be correct in that case.  But that's more a feature request, and could revisit saying 'initial' then- because otherwise, I lean toward Yuri's take.